app/src/main/res/layout/activity_bmi_calculator.xml

```diff
@@ -45,7 +45,7 @@
                     android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:gravity="center_horizontal"
-                    android:text="Height"
+                    android:text="@string/height"
                     android:textSize="24sp"/>
```

Comment:
Fixed the error, and added the missing strings.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg
https://www.youtube.com/watch?v=YaWmNhlOXYk